includes/wikia/models/DesignSystemSharedLinks.class.php

```diff
@@ -80,7 +80,7 @@ private function getLangWithFallback( $lang ) {
 			'games' => 'http://fandom.wikia.com/games',
 			'movies' => 'http://fandom.wikia.com/movies',
 			'tv' => 'http://fandom.wikia.com/tv',
-			'fandom-university' => 'http://community.wikia.com/wiki/Wikia_University',
+			'fandom-university' => 'http://community.wikia.com/wiki/Fandom_University',
 			'user-signin' => 'https://www.wikia.com/signin',
 			'user-register' => 'https://www.wikia.com/register',
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "user-register" 11 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)